app/app.js

```diff
@@ -39,11 +39,19 @@ const getPactVersion = (json) => {
     }
 }
 
+function isValidVersion(version) {
+    return /^\d+\.\d+\.\d+$/.test(version);
```

Comment:
Out of curiosity, is this enough to resolve the vulnerability?

Reply:
Hmm, my reasoning was that this would prevent `require()` potentially being called with suspicious `version` strings. Not sure how Snyk will assess the change; unfamiliar with how to understand this before merging but that would be a nice capability 🤔 .

Reply:
I guess we'll see when it gets merged.

Reply:
If this turns out not to be enough… are you thinking we may need parse the value to satisfy the check, _instead of just checking it as I'm doing here_?

i.e. `const safeVersion = sanitise(version);`

Reply:
Potentially, but unsure.

Reply:
I'm also curious to see if the tool is smart enough to understand this fix, another way to fix this is using a `switch` on version like the following:
```
switch (version) {
  case '2.0.0': return require(`./parsers/2.0.0/pact-parser`);
  ...
}
```